src/main/java/com/checkmarx/flow/custom/ServiceNowTracker.java

```diff
@@ -334,10 +340,10 @@ public String getIssueKey(Issue issue, ScanRequest request) {
     @Override
     public String getXIssueKey(ScanResults.XIssue issue, ScanRequest request) {
         if(flowProperties.isTrackApplicationOnly() || ScanUtils.empty(request.getBranch())){
-            return String.format("%s @ %s", issue.getVulnerability(), issue.getFilename());
+            return String.format(ScanUtils.ISSUE_TITLE_KEY, request.getProduct().getProduct(), issue.getVulnerability(), issue.getFilename());
```

Comment:
This and the line below are incomplete changes, you have to take SCA into account here, the code for this method should be similar to the one in GitHubTracker and GitlabTracker, like this:

```java
       if(flowProperties.isTrackApplicationOnly() || ScanUtils.empty(request.getBranch())){
            return String.format(ScanUtils.ISSUE_TITLE_KEY, request.getProduct().getProduct(), issue.getVulnerability(), issue.getFilename());
        }
        else {
            return ScanUtils.isSAST(issue)
                    ? String.format(ScanUtils.ISSUE_TITLE_KEY_WITH_BRANCH, request.getProduct().getProduct(), issue.getVulnerability(), issue.getFilename(), request.getBranch())
                    : ScanUtils.getScaSummaryIssueKey(request, issue);
        }
```